### ISSUE TITLE
trying to show region labels causes "e is undefined"

### ISSUE THREAD

**Issue by emko** [themustafaomar/jsvectormap#92]
when i try to show label regions i get this error.

```
Unhandled Runtime Error

TypeError: e is undefined
Source
getLabelOffsets
node_modules/jsvectormap/dist/js/jsvectormap.min.js (1:8925)
e
node_modules/jsvectormap/dist/js/jsvectormap.min.js (1:9600)
_createRegions
node_modules/jsvectormap/dist/js/jsvectormap.min.js (1:18562)
V</e._init
node_modules/jsvectormap/dist/js/jsvectormap.min.js (1:27767)
t
node_modules/jsvectormap/dist/js/jsvectormap.min.js (1:27387)
t
node_modules/jsvectormap/dist/js/jsvectormap.min.js (1:32620)
```

**Comment by dgillier** [comment]
Hello,
I'm trying to display custom region label, I'm having this error, I don't know if it's related to what @emko is talking about.

I'm using this, as instructed on the doc:
```
            labels: {
                regions: {
                    render(code) {
                       return ['BE', 'FR'].indexOf(code) > -1 ? 'Hello' + code : null

                    }
                }
            },
```


```
Uncaught TypeError: Cannot read properties of undefined (reading 'offsets')
    at e.getLabelOffsets (jsvectormap.min.js:1:8953)
    at new e (jsvectormap.min.js:1:9609)
    at t._createRegions (jsvectormap.min.js:1:18692)
    at e._init (jsvectormap.min.js:1:27925)
    at jsvectormap.min.js:1:27695
```

**Comment by themustafaomar** [comment]
This issue has been fixed in https://github.com/themustafaomar/jsvectormap/commit/78b1481edc68194ab038330c3853285aee9fe1b0 and it will be available on the next version, v1.5.1 will be available by the end of the week.

Thanks!

**Comment by dgillier** [comment]
Thanks a lot !
